docroot/sites/sppa.uiowa.edu/modules/sppa_core/sppa_core.module

```diff
@@ -7,6 +7,181 @@
 
 use Drupal\Core\Entity\ContentEntityInterface;
 use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Custom node content type form defaults.
+ */
+function _sppa_node_form_defaults(&$form, $form_state) {
```

Comment:
instead of duplicating _sitenow_node_form_defaults, you can call this and override after if necessary. like here https://github.com/uiowa/uiowa/blob/main/docroot/profiles/custom/sitenow/sitenow.profile#L509

Reply:
@joewhitsitt thank you so much! So would I need to create a `sitenow_private_pages.settings_form` then for the featured_image_display settings? I get an error when calling the function.

Reply:
ah i see... that stuff wasn't in there when i first created it. ideally it would be adjusted to be more flexible so it isn't following the `sitenow_` prefix. let me ponder this a bit

Reply:
Okay! Thank you so much for your help!

Reply:
I am going to fix the logic around the `field_featured_image_display` stuff in this function. plan on `sppa_core.settings_form` and you can comment out this snippet for now.